Source/MDKWhitelistExtractor/ExtractorPlugin.cs

```diff
@@ -142,7 +142,7 @@ Type FindInterface(Type block)
 
         IEnumerable<Type> FindBlocks(Assembly gameAssembly, HashSet<AssemblyName> visitedAssemblies = null)
         {
-            visitedAssemblies ??= new HashSet<AssemblyName>(new AssemblyNameComparer());
+            visitedAssemblies = visitedAssemblies ?? new HashSet<AssemblyName>(new AssemblyNameComparer());
```

Comment:
Please don't change code just because you don't _like_ it.

Reply:
Actually, for some reason my VS2017 failed to compile the short form (syntax error).
I changed it, and then promptly forgot to change it back before sending PR.

Reply:
I wouldn't be surprised if VS2017 doesn't support the C# version.

Reply:
Or; maybe the project is set to use the latest default. VS2017 interprets that differently than VS2019